clients/sender/controllers/main.html

```diff
@@ -1 +1,6 @@
-<h3>This is the main page</h3>
+<h3>Card Cast</h3>
+
+<form method="get" action="sendMessage()">
+  <textarea id="textBox" rows="10" cols="80">Your markdown here</textarea>
```

Comment:
This needs to be tied to the controller `$scope` using `ng-model`. It can be named whatever, `$scope.data`... `$scope.message`... and that is what should be used in the `sendMessage` function.